find-files.nix

```
@@ -242,7 +242,10 @@ rec {
         for
           (guard (toLower section == "core" && toLower key == "excludesfile"))
           (_:
-            resolveFile (home /.) value
+            # SAFETY: Ultimately this path will get passed into
+            # `builtins.readFile`, which ignores the context anyways. The
+            # context is only relevant if it becomes part of a derivation.
```

`readFile` does realise the context, and resolves CA outputs (`ca-derivations`), so this isn't quite true.

Maybe something like this, though it could be more detailed.

```suggestion
            # SAFETY: In some circumstances, `value` carries a string context,
            # which would result in an error in `resolveFile (home /.)`.
```

---

Ah! I had tested out that the output of `readFile` doesn't  change depending on the context of its input, but I didn't think about the mechanics of how it gets the result.

Let me know if the new comment is okay.